reactioncommerce/reaction
# Convert to react: Tag navigation

Update: 22/05
Resolves #2043 TagNav React Conversion



**Things to Test:**

1.  TopLevel Tags List
    * Adding a new Tag 
        * Type into the tag box, save by clicking "enter". Also save by clicking the "+" icon
    * Drag tags around to re-order them
    * Delete tags by clicking the "x" icon
    * Add more tags

2.  Second Level Tags (TagTree)
    * In Edit mode, click the drop caret beside a TopLevel tag to expand dropdown
    * Add header tags
    * Add body tags under those header tags
    * Confirm you can edit the tags and can delete (with the "x" icon)

3.  Drag & Drop
    * Drag to re-order tags under a tagTree header
    * Dragging the tagTree header should lift all subtags under it (in-prog)

4.  Related feature to check
    * Product tags: TagNav share components with Product tags... check to confirm product tag functionality is still intact.

5.  Confirm that these bugs are fixed
    * #1312
    * #1313

Pending issue: On mobile, clicking the hamburger icon doesn't show the tagNav. Being worked on
